satpy/tests/reader_tests/__init__.py

```diff
@@ -37,9 +38,8 @@
                                       test_safe_sar_l2_ocn, test_viirs_edr_active_fires,
                                       test_hdfeos_base, test_modis_l2,
                                       test_electrol_hrit, test_mersi2_l1b,
-                                      test_avhrr_l1b_gaclac, test_vaisala_gld360,
-                                      test_fci_l1c_fdhsi, test_tropomi_l2,
-                                      test_hsaf_grib)
+                                      test_avhrr_l1b_gaclac, test_fci_l1c_fdhsi,
+                                      test_viirs_gran)
```

Comment:
You accidentally removed a lot of these readers from the tests (see diff on github). The `test_vaisala_gld360`, `test_tropomi_l2`, and `test_hsaf_grib` reader tests have been removed from the import and the test suite at the bottom.

Reply:
Yikes! Added back now